Documentation/Docker and PostgreSQL.md

```diff
@@ -1,22 +1,16 @@
-# Docker - MongoDB Server
-* Will use a docker MongoDB image and run that image as a container to start our MongoDB server for ATC-Backend
-    * backend will then connect to this MongoDB server instance via localhost:port
+# Docker - PostgreSQL Server
+* IMPORTANT NOTE: You need the database to be running in Docker constantly while you are developing (coding) the software. You may also need to change your DATABASE_URL .env variable to this URL: `postgresql://postgres:postgres@localhost:5003/postgres`
+  
 
 * **SCRIPTS**
+  *FOR ALL SCRIPTING, USE GIT BASH INSTEAD OF POWERSHELL*
```

Comment:
I think we may just want to remove all references to these scripts since they no longer exist in our repo. Instead we may want to instruct them to run something like `npx prisma migrate reset` for initial db setup.

@pariahGH

Reply:
I know why you added that note about sometimes needing to use bash instead of powershell to run scripts on windows @FegelSamuel. Do you remember which command it was that we needed this for?

Reply:
Just so I remember to actually write this down, you mentioned to me that it's best practice to use Git Bash for all scripting in general.

Reply:
I wouldn't say best practice, that depends on who you ask. I would say it's more just my personal preference.